sites/website/sidebars.js

```diff
@@ -7,9 +7,10 @@ module.exports = {
         },
         {
             type: "category",
-            label: "Using FAST Components",
+            label: "Using the Components",
```

Comment:
Could "the" be removed also?  Doesn't seem necessary.

Reply:
Well, it's specifically about using our components not just using any set of components. So, I think it should stay for now.